src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -32,60 +32,60 @@ export
 #
 ########################################################################
 
+# TODO: eliminate this function again
 """
     submatrix(A::MatElem{T}, i::Int, j::Int, m::Int, n::Int)
 
 Return the `m x n` submatrix of `A` rooted at `(i,j)`
 """
-# TODO: eliminate this function again
```

Comment:
Please remove these changes in this file (they are necessary to use the docstrings, of course, but they'll be removed soon anyway, and changing this now just introduces a merge conflict down the road)